bindings/python/crocoddyl/deprecation.py

```diff
@@ -0,0 +1,8 @@
+from . import libcrocoddyl_pywrap as _crocoddyl
+from .deprecated import deprecated, DeprecatedWarning
+
+
+class ActivationModelSmoothAbs(_crocoddyl.ActivationModelSmooth1Norm):
+    @deprecated("crocoddyl::ActivationModelSmoothAbs is deprecated: Use ActivationModelSmooth1Norm")
+    def __init__(self, nr, eps=1.):
```

Comment:
Since you are inheriting, you don't need to redefine the `__init__` function.

Reply:
I do in order to include the @deprecated decorator.
As far I know, this is the approach to be taken for deprecating a class in Python.

Reply:
I see. Thanks for the explanation